### DISCUSSION TITLE
Can it support 32bit original artwork

### DISCUSSION THREAD

**Discussion by dbghex** [rustdesk/rustdesk#5087] [Feature Request]
Can it support 32bit original artwork

**Comment by dinger1986** [comment]
You can increase the framerate dramatically so get really good quality , worth playing with RustDesk and seeing if it fits your needs

**Comment by dbghex** [comment]
I compared todesk, RDP, rustdesk (maximum bit rate), todesk, RDP picture clarity is very good, rustdesk is the worst, I would like to ask whether there is a better picture quality for users to choose
